### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by aileo**
PR for #4

### CODE REVIEW

**Comment by aileo** [comment]
@JohnBerd, I am curious about your view on this subject.
Could you have a look and tell me what you think about this part of the lib's API ?

File: README.md [hunk 1]
```diff
@@ -191,6 +191,39 @@ This method enable access to pages defined with `authenticated=true`.
...
+#### setLocale
+
+Load locale file from URL and re-render.
+
+```js
+import oiler from 'oiler';
+
+await oiler.setLocale('fr');
+```
+
```
Comment (JohnBerd) [inline README.md:222]: Maybe add an example of the usage of this feature in JSX files (eg: <p>{oiler.text(['what', 'content'])}</p>)

**Review by JohnBerd — COMMENTED** [review]
The abstraction is so clear that it is very easy for a developper to implement any new language in any app. To me, it is the best way you could implement it.

File: README.md (hunk 1 above)
Comment (aileo) [reply README.md:222]: Good point

**Pull request merged**